### PR TITLE
Use AccentColor as background for selected text in the text layer (bug 1790309)

### PR DESCRIPTION
**Pull request by calixteman**
and use the default color for the viewer in general.

### CODE REVIEW

File: web/xfa_layer_builder.css [hunk 1]
```diff
@@ -59,7 +59,7 @@
 }
 
 .xfaLayer ::selection {
-  background: rgba(0, 0, 255, 1);
+  background: AccentColor;
```
Comment (Snuffleupagus) [inline web/xfa_layer_builder.css:62]: The pre-processor suggestion, see above, applies here as well.

---

Also, is there not an `opacity` value that needs to be changed in this file as well?

File: web/text_layer_builder.css [hunk 2]
```diff
@@ -21,7 +21,7 @@
   right: 0;
   bottom: 0;
   overflow: hidden;
-  opacity: 0.2;
+  opacity: 0.3;
```
Comment (Snuffleupagus) [inline web/text_layer_builder.css:24]: Also, how about using `0.25` as a compromise between the old/new value such that this works OK with both color-values (given the pre-processor comments elsewhere)?

**Comment by Snuffleupagus** [comment]
It looks like the last update (I guess accidentally?) reverted the fixes to comments https://github.com/mozilla/pdf.js/pull/15461#discussion_r974527143 and https://github.com/mozilla/pdf.js/pull/15461#discussion_r974527143.

Also, is text-selection looking correct in the xfaLayer even against the `master` branch given that there's no opacity used there?

**Comment by calixteman** [comment]
About the xfa layer: yes the result is not really nice and consistent.
We can't really set the opacity since the text is not in a separate layer.
Anyway, I removed those settings in the xfa layer and let the usual selection colors: it's still inconsistent but.

**Review by Snuffleupagus — APPROVED** [review]
Seems to work fine in Firefox Nightly!

**Comment by calixteman** [comment]
> Seems to work fine in Firefox Nightly!

Yep and it works fine on mac.

**Pull request merged**